mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -506,18 +506,17 @@ def test_spectral_connectivity_time_resolved(method, mode):
 
     # run connectivity estimation
     con = spectral_connectivity_time(
-        data, freqs=freqs, method=method, mode=mode)
-    assert con.shape == (n_epochs, n_signals * 2, n_freqs, n_times)
+        data, cwt_freqs=freqs, method=method, mode=mode)
+    assert con.shape == (n_epochs, n_signals * 2, len(con.freqs))
```

Comment:
I'm not 100% following what's going on here. The time dimension is not present anymore it seems?

Reply:
The time dimension is omitted, as the conenctivity scores are by definition averaged over time. Importantly, the sum should be taken before the absolute value for PLV since absolute value is not linear for complex numbers. See Eq 2. in [Bruna et al., 2018](https://iopscience.iop.org/article/10.1088/1741-2552/aacfe4/pdf).

Reply:
I see, so in essence you either take a sliding window and compute time-resolved spectral conn (i.e. each epoch/window has one connectivity matrix), or you take disparate epochs of some time period over a task and compute connectivity that can track things over time in that specified period (i.e. connectivity over time of a specific task).

I see. I think I keep on getting confused when I don't think about it for awhile, the difference between `spectral_connectivity_time` and `spectral_connectivity_epochs`. I think adding some sort of notes describing this to each docstring will eventually be useful. If you're up for it? Otw, I can add it in a downstream PR.

Reply:
I could try adding some notes to the docs. However, one thing that's not clear to me is how the time dimension disappears in `spectral_connectivity_epochs` when using the multitaper method. I tried to follow the code, but didn't quite keep up with the shapes of arrays at each point.